### PR TITLE
Fixed documentation

### PR DESCRIPTION
**Pull request by jctim**
Fixed:
* typo
* indents
* code highlighting

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2786)

### CODE REVIEW

File: docs/developers.rst [hunk 1]
```diff
@@ -11,20 +11,20 @@ Repositories
 ^^^^^^^^^^^^
 The beam repository on github `is here. <https://github.com/LBNL-UCB-STI/beam>`_
 
-The convention for merging into the master branch is that master needs to be pass all tests and at least one other active BEAM developer needs to review your changes before merging. Please do this by creating a pull request from any new feature branches into master. We also encourage you to create pull requests early in your development cycle which gives other's an opportunity to observe and/or provide feedback in real time. When you are ready for a review, invite one or more through the pull request. 
+The convention for merging into the master (or develop) branch is that master (or develop) needs to be pass all tests and at least one other active BEAM developer needs to review your changes before merging. Please do this by creating a pull request from any new feature branches into master (or develop). We also encourage you to create pull requests early in your development cycle which gives other's an opportunity to observe and/or provide feedback in real time. When you are ready for a review, invite one or more through the pull request.
```
Comment (jctim) [inline docs/developers.rst:14]: A clarification needed regarding master/develop

Comment (jctim) [reply docs/developers.rst:14]: Hi @JustinPihony,

Could you suggest whether I should leave 'master' branch as a base or 'develop' in this concrete part?

As I know the 'develop' branch is used as a base development branch and the process regarding merging to 'develop' is clear for me and corresponds to what is described here
But I'm not familiar with a process when we merge develop (or anything else) to master

**Comment by jctim** [comment]
Test!

File: docs/developers.rst (hunk 1 above)
Comment (JustinPihony) [reply docs/developers.rst:14]: @jctim Sorry for the delay - Yes, this convention is now `develop`. `master` is only for when we have releases - this is something that most developers will do and likely will not need to be documented at this time.

**Comment by JustinPihony** [comment]
test!

**Comment by JustinPihony** [comment]
test!

**Pull request merged**